### DISCUSSION TITLE
how do i can reset node web gui password ?

### DISCUSSION THREAD

**Discussion by panayotovip** [mysteriumnetwork/node#4962] [Q&A]
how do i can reset node web gui password ?
when i try to use "FORGOT PASSWORD?" it open address :
https://docs.mysterium.network/node-runners/troubleshooting#forgot-password
and load "Page Not Found".
is it something wrong with the system ?

**Comment by adinetech** [comment]
We updated the documentation, so old links will no longer work. Please use https://docs.mysterium.network/troubleshooting/forgot-password
